source/administrator-guide/managing-metadata-standards/metadata-identifier.rst

```diff
@@ -0,0 +1,47 @@
+.. _metadata_identifier:
+
+Configuring metadata identifier
+###############################
+
+
+Some organisations require to define the metadata identifiers using URN templates.
```

Comment:
Some organisations require the definition of metadata identifiers using URN templates (should probably expand URN)